server/controller/employee/addNewEmployee.js

```diff
@@ -0,0 +1,15 @@
+const postNewEmployee = require('../../database/query/employee/postNewEmployee');
+
+const addNewEmployee = (req, res) => {
+	const employeeData = req.body;

```

Comment:
I think you should make sure and validate the data before pass them to the query